qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider_for_plugin_providers.h

```diff
@@ -35,9 +35,6 @@
 
 #include "plugin_provider.h"
 #include "ros_pluginlib_plugin_provider.h"
-#include "exports.h"
-
-template class QT_GUI_CPP_DECL qt_gui_cpp::RosPluginlibPluginProvider<qt_gui_cpp::PluginProvider>;
```

Comment:
Can it be conditionally guard by `_MSC_VER` for the fix? I know you want to unblock `armhf` build, but I am worried removing that will cause Windows build break too. I will need to take a look how to make this part more portable...

Reply:
Your PR added the definitions in a separate `.cpp` file. Isn't that enough? Please try to compile this patch on Windows and comment here as soon as feasible.

Depending on how long that might be I am tempted to release this (or even revert the PR introducing the regression) to unblock armhf. If Windows is being broken that is unfortunate but secondary since until that PR it didn't work in the first place.

Reply:
@dirk-thomas, I did quick test to remove it and build `qt_gui_cpp` and `rqt_gui_cpp` together. Surprisingly it didn't show any build issues. Please go ahead to release this fix. I will also keep eyes on tonight CI windows builds from our side too.